plugin/rapidfire.vim

```diff
@@ -3,6 +3,4 @@ if exists('g:rapidfire_loaded')
 endif
 let g:rapidfire_loaded = 1
 
-command! -nargs=1
-      \ Rapidfire
-      \ call rapidfire#command#rapidfire#call(<q-mods>, <f-args>)
+command! -nargs=1 Rapidfire call rapidfire#call(<q-mods>, <f-args>)
```

Comment:
Do you have any rules about vim script line breaks?

Reply:
No. I don't have any rules or favor or whatever. But some people do for example, some people don't like the default indent level for line breaks so they changed.